### PR TITLE
MRG Extend tutorials - HED epochs, evoked plots and arithmetic

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
Adding a bit to the tutorials.

@agramfort this what you were thinking about?

### CODE REVIEW

**Comment by Eric89GXL** [comment]
I hope merging the other PR doesn't make this rebase too bad

File: tutorials/plot_eeg_erp.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 of EEG, namely:
     - setting the reference
     - using standard montages :func:`mne.channels.Montage`
+    - Evoked arithmetics (e.g. differences)
```
Comment (Eric89GXL) [inline tutorials/plot_eeg_erp.py]: just "arithmetic"

**Comment by Eric89GXL** [comment]
Thanks for tackling this

File: tutorials/plot_eeg_erp.py (hunk 1 above)
Comment (jona-sassenhagen) [inline tutorials/plot_eeg_erp.py]: Is that a british/american thing? :)

File: tutorials/plot_eeg_erp.py [hunk 2]
```diff
@@ -128,3 +129,62 @@
...
+
+# If they are stored in a list, they can be easily averaged, for example,
+# for a grand average across subjects (or conditions).
+grand_average = mne.grand_average(all_evokeds)
+mne.write_evokeds('/tmp/tmp-ave.fif', all_evokeds)
+
+# If Evokeds objects are stored in a dictionary, they can be retrieved by name.
+
+all_evokeds = dict((cond, epochs[cond]) for cond in event_id)  # dict comp.
+print(all_evokeds["auditory/left"])
```
Comment (dengemann) [inline tutorials/plot_eeg_erp.py]: what does this actually add? Maybe then showing some plot where this is useful would be cool. Maybe from our Frankfurt workshop, I think we had a use case for this, like a self made roi plot.

File: tutorials/plot_epoching_and_averaging.py [hunk 3]
```diff
@@ -141,6 +142,16 @@
...
 
 ###############################################################################
+# Notice we have used forward slashes ('/') to separate the factors of the
+# conditions of the experiment. We can use these 'tags' to select for example
+# all left trials (both visual left and auditory right) ...
+
+epochs_left = epochs['Left']
+
+# ... or to select a very specific subset. This is the same as above:
+evoked_left = epochs['Auditory', 'Left'].average(picks=picks)
```
Comment (dengemann) [inline tutorials/plot_epoching_and_averaging.py]: surprised this works .. but it is an implicit tuple and we support tuples. interesting.

**Comment by dengemann** [comment]
@jona-sassenhagen yeah thanks, this goes exactly in the direction we had in mind.

File: tutorials/plot_eeg_erp.py (hunk 2 above)
Comment (jona-sassenhagen) [inline tutorials/plot_eeg_erp.py]: We recently had an issue where some people expressed they already did this, and others were grateful of being shown that option. I think having semantically accessible evokeds is clearly useful! But I'll add an example.

The Frankfurt example is probably too elaborate though.

**Comment by jona-sassenhagen** [comment]
Should be ready. Merge if you're satisfied.

**Comment by agramfort** [comment]
thx @jona-sassenhagen

**Comment by jona-sassenhagen** [comment]
Alex tell me if you think I should go over any other part of the tutorials!

**Comment by agramfort** [comment]
please do in another PR when you have time.

**Comment by jona-sassenhagen** [comment]
Yes, but I mean, what part of the tutorials?

**Comment by agramfort** [comment]
all ? :)

**Comment by jona-sassenhagen** [comment]
Haha okay. I'll go over them once they're on the website, for 0.13.

**Comment by agramfort** [comment]
https://mne-tools.github.io/dev/tutorials.html should be up to date.

it's updated automatically by circle ci

I'll update martinos.org when I have time

**Comment by agramfort** [comment]
updated

http://martinos.org/mne/dev/tutorials.html

**Pull request merged**